guides/v3.9.0/tutorial/service.md

```diff
@@ -44,6 +44,13 @@ Next, add your new API key to the application by stopping the server and restart
 LEAFLET_MAPS_API_KEY=<your key here> ember s
 ```
 
+NOTE - If you are working on windows machine, use the set command
+
+```bash
+set LEAFLET_MAPS_API_KEY=<your key here>
```

Comment:
I think trying to accommodate  every possible shell might not be realistic. For instance, I use `fish` and I have to prefix setting of env variables with `env`. I'm not sure if we want to have specific commands here, as `bash` is typically the standard.

Reply:
The difference here is that if you are customizing your shell, you should already have the knowledge on how to translate instructions to it. That might not be true for someone using the default shell of their OS, so I think this callout is useful!

Reply:
I think it might be a little bit vague for that though. You could be on a Windows machine and using any shell. If we want to provide both bash and command prompt style commands, we can, but I would tend to prefer saying something like `Note: all commands are written for bash and may or may not work on your shell` something like that maybe.

Reply:
All comments from both of you sounds absolutely perfect and fitting the right context ! As requested by @locks I will make the necessary tweaks

Reply:
Please look through the changes, I have mentioned that primarliy we go with bash commands as the standard but for default OS command prompts we have added the case